src/core/views/components/iframe.js

```diff
@@ -23,14 +23,16 @@ class Iframe extends Component {
 
   // Fixes scroll not responding in renderer v1
   handleLoad () {
-    this.iframeRef.style.height = `${this.iframeRef.offsetHeight + 1}px`
-    setTimeout(() => {
-      this.iframeRef.style.height = ''
-      applyIOSFooterHack(this.iframeRef)
-      applyIOSIframeResizeHack(this.iframeRef)
-
-      this.props.onLoad && this.props.onLoad(this.iframeRef)
-    }, 1)
+    if (this.iframeRef) {
```

Comment:
how about 
```typescript
if (!this.iframeRef) { return }
// ... the rest
```
?
In most cases, such defensive programming makes code more readable, since it becomes less nested